azure/functions/timer.py

```diff
@@ -5,19 +5,29 @@
 import typing
 
 from azure.functions import _abc as azf_abc
-
 from . import meta
 
 
 class TimerRequest(azf_abc.TimerRequest):
```

Comment:
@YunchuWang - Getting your input here - If we plan to add PyDocs here - what could we add here?

@pdthummar No need to worry in this PR (mostly). But we should explore adding more PyDoc comments as they eventually show up in [azure.functions.timerrequest](https://docs.microsoft.com/en-us/python/api/azure-functions/azure.functions.timerrequest?view=azure-python)

Reply:
@vrdmr  would suggest giving a overview what this class does, adding definition of each field in init, and link to https://docs.microsoft.com/en-us/azure/azure-functions/functions-bindings-timer?tabs=in-process&pivots=programming-language-java#attributes